app/aws/s3.py

```diff
@@ -207,9 +204,8 @@ def read_s3_file(bucket_name, object_key, s3res):
                 extract_personalisation(job),
             )
 
-    except LookupError:
-        # perhaps our key is not formatted as we expected.  If so skip it.
-        current_app.logger.exception("LookupError #notify-debug-admin-1200")
+    except Exception:
+        current_app.logger.exception("Exception")
```

Comment:
Would it be more helpful if we actually got the error to see what was going on?

```suggestion
    except Exception as e:
        current_app.logger.exception(str(e))
```

Or would it not provide useful information and just clutter the logs?